Update mpc-qt_ja.ts

coolvitto opened the pull request.

tbertels commented:
The `vanished` strings don't show up in Weblate because they're old string (meaning not being used anymore in mpc-qt).
They're kept so that translators don't have to retranslate them in case they're used again in the future.

coolvitto commented:
I want to check how the app displays Japanese, so I'll leave it up for a while.
I think I'll probably find something to fix.

tbertels commented:
No problem, it'll ship in the next release (June-July) so there's still plenty of time.

coolvitto commented:
By the way, is there any way to check the compiled "mpc-qt_ja.qm" locally?

tbertels commented:
If you build mpc-qt, the .ts file will be converted to .qm and added to the mpc-qt binary.

You can build it automatically on GitHub by forking this repository (which you've already done automatically) and then pushing your updates [to it](https://github.com/coolvitto/mpc-qt) by using a git client, for example [the one from GitHub](https://github.com/apps/desktop).

The Windows build will then appear in [your Actions](https://github.com/coolvitto/mpc-qt/actions) in the Msys2 workflow runs as an artifact.
Here's an example on this repository: https://github.com/mpc-qt/mpc-qt/actions/runs/13724764760

coolvitto commented:
Thank you!!😀
I'll check the Japanese translation again using this executable file.
________________________________
差出人: Thomas Bertels ***@***.***>
送信日時: 2025年3月9日 2:41
宛先: mpc-qt/mpc-qt ***@***.***>
CC: coolvitto ***@***.***>; Author ***@***.***>
件名: Re: [mpc-qt/mpc-qt] Update mpc-qt_ja.ts (PR #367)


If you build mpc-qt, the .ts file will be converted to .qm and added to the mpc-qt binary.

You can build it automatically on GitHub by forking this repository (which you've already done automatically) and then pushing your updates to it<https://github.com/coolvitto/mpc-qt> by using a git client, for example the one from GitHub<https://github.com/apps/desktop>.

The Windows build will then appear in your Actions<https://github.com/coolvitto/mpc-qt/actions> in the Msys2 workflow runs as an artifact.
Here's an example on this repository: https://github.com/mpc-qt/mpc-qt/actions/runs/13724764760

—
Reply to this email directly, view it on GitHub<https://github.com/mpc-qt/mpc-qt/pull/367#issuecomment-2708412129>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BGH4IK2G26VU4IDBE3XZBHL2TMTWFAVCNFSM6AAAAABYQWOXXCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMBYGQYTEMJSHE>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

[tbertels]tbertels left a comment (mpc-qt/mpc-qt#367)<https://github.com/mpc-qt/mpc-qt/pull/367#issuecomment-2708412129>

If you build mpc-qt, the .ts file will be converted to .qm and added to the mpc-qt binary.

You can build it automatically on GitHub by forking this repository (which you've already done automatically) and then pushing your updates to it<https://github.com/coolvitto/mpc-qt> by using a git client, for example the one from GitHub<https://github.com/apps/desktop>.

The Windows build will then appear in your Actions<https://github.com/coolvitto/mpc-qt/actions> in the Msys2 workflow runs as an artifact.
Here's an example on this repository: https://github.com/mpc-qt/mpc-qt/actions/runs/13724764760

—
Reply to this email directly, view it on GitHub<https://github.com/mpc-qt/mpc-qt/pull/367#issuecomment-2708412129>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BGH4IK2G26VU4IDBE3XZBHL2TMTWFAVCNFSM6AAAAABYQWOXXCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMBYGQYTEMJSHE>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

coolvitto commented:
I launched the app, checked the Japanese translation and made some corrections, so could you please create a build for testing like you did last time?(https://github.com/mpc-qt/mpc-qt/actions/runs/13724764760)

tbertels commented:
You don't need to (and shouldn't) make a pull request here (#369) to trigger a new build with your translation.
You just need to open a pull request on your own fork, [like I just did](https://github.com/coolvitto/mpc-qt/pull/1).
If you approve the workflow run in the pull request, the Windows build will appear [in the Actions of your fork](https://github.com/coolvitto/mpc-qt/actions) under Msys2.

coolvitto commented:
I have translated it, but I have never done such an operation, so I do not fully understand it.
Does "my fork" refer to the following screen?
![sshot-1](https://github.com/user-attachments/assets/2a5d240f-8444-43a9-834c-896cbce66429)
What should I do on this screen?

coolvitto commented:
I didn't really understand what I was doing, but after fiddling around with it a bit, I finally managed to do it.
![sshot-7](https://github.com/user-attachments/assets/d00320bd-0253-4e59-a22a-34bdd8460b13)

tbertels commented:
>What should I do on this screen?

I was expecting GitHub to show you a button to enable the workflow run, but for some reason it doesn't.
But since you managed to push commits, it doesn't matter.
